### ISSUE TITLE
Support mounting multiple volumns

### ISSUE THREAD

**Issue by sophia-guo** [adoptium/aqa-tests#3450]
[function parse_docker_args() ](https://github.com/adoptium/aqa-tests/blob/master/external/external.sh) parses parameters for docker build and docker run command.  Currently it parses parameter by `-v` and `-i`. `-v` is used for docker run command to mount volumes, which could be multiples.

The function can be updated to support multiple volumes, which can be tested by test [quarkus_quickstarts_test](https://github.com/adoptium/aqa-tests/blob/master/external/quarkus_quickstarts/playlist.xml)

**Comment by abhijeetjejurkar** [comment]
will this code work?, If Yes, will you please Assign me this?

`function parse_docker_args() {
# parse docker_args to two variable: mountV and  imageArg
	while [[ $# -gt 0 ]] && [[ ."$1" = .-* ]] ; do
		opt="$1";
                c=""
		shift; 

		case "$opt" in
			"--volume" | "-v" )
                               if [ $# -gt 1 ]; 
                               then
                                      while [ $# -gt 0 ]
                                      do
                                             c+="-v $1 ";
                                             shift 2;
                                       done      
                                      mountV=c;
                                             
                               else
				       mountV="-v $1";
				       shift;;
			"--image" | "-i" )
				imageArg="$1"; 
				shift;;
			*) echo >&2 "Invalid docker args option: ${opt}"; exit 1;
		esac
	done
}`

**Comment by sophia-guo** [comment]
There is no need to do the inner while loop in side of case `-v` as multiple ` -v` could be in any order. All `-v` options can just be appended to one. Like

```
function parse_docker_args() {
# parse docker_args to two variable: mountV and  imageArg
	while [[ $# -gt 0 ]] && [[ ."$1" = .-* ]] ; do
		opt="$1";
		shift; 
                 mountV=""  # initial empty
		case "$opt" in
			"--volume" | "-v" )
				mountV="${mountV} -v $1";     # append
				shift;;
			"--image" | "-i" )
				imageArg="$1"; 
				shift;;
			*) echo >&2 "Invalid docker args option: ${opt}"; exit 1;
		esac
	done
}
```

**Comment by sophia-guo** [comment]
You can also create a shell script to test it.

**Comment by abhijeetjejurkar** [comment]
Thankyou, I have not set-up Project on my system yet, I will try to setup project on my environment

**Comment by sophia-guo** [comment]
`mountV=""  # initial empty` should be outside of while loop.

**Comment by abhijeetjejurkar** [comment]
Yeah 👍 , else in every iteration of while loop `mountV` will becoming empty string

**Comment by abhijeetjejurkar** [comment]
Will you please assign me this issue?
